.github/workflows/ci.yml

```diff
@@ -121,6 +124,10 @@ jobs:
                   DEPLOY_ENV=ci
                   echo "DEPLOY_ENV=$DEPLOY_ENV" | tee -a $GITHUB_ENV
 
+            - name: Wait if another CI AWS already stack exists
+              run: |
+                  timeout 2h ./wait-for-stack-teardown.bash geospatial-data-lake-ci
```

Comment:
@l0b0 , can you add check for `geospatial-data-lake-networking-ci` and `geospatial-data-lake-processing-ci` stacks once `wait-for-stack-teardown.bash` can support multiple stacks ?

Reply:
It already supports multiple stacks, just pass any number of stack names to the script and it'll wait for all of them to be torn down. But neither of those stacks are currently being torn down by CI, so I don't quite understand.

Reply:
I see it does support multiple stacks, that's why I am saying that it might be good to add `geospatial-data-lake-networking-ci` and `geospatial-data-lake-processing-ci`

Reply:
But if we add those two this task will always fail, because at least the `geospatial-data-lake-networking-ci` stack isn't torn down by CI.

Reply:
Sorry, only `geospatial-data-lake-processing-ci` needs to be added.

Reply:
Done :)